modules/@apostrophecms/template/index.js

```diff
@@ -1314,11 +1314,14 @@ module.exports = {
           // Annotate each individual widget with its options
           // Each widget must elect into this by creating an
           // `annotateWidgetForExternalFront() method.
-          const manager = self.apos.area.getManager?.(item.type) ||
-            self.apos.area.widgetManagers?.[item.type];
-
-          const widgetOptions = manager.annotateWidgetForExternalFront() || {};
```

Comment:
The default implementation returns `{}` and it's a new method so there should be no need for `||` here